### ISSUE TITLE
Custom heads create errors

### ISSUE THREAD

**Issue by Flexo013** [Multiverse/Multiverse-Inventories#266]
#### Reproduce
1. Spawn a custom head without a user. For example: `/give @p skull 1 3 {display:{Name:"Bread"},SkullOwner:{Id:"a75e3f60-2242-4429-8ece-bcde7753b064",Properties:{textures:[{Value:"eyJ0ZXh0dXJlcyI6eyJTS0lOIjp7InVybCI6Imh0dHA6Ly90ZXh0dXJlcy5taW5lY3JhZnQubmV0L3RleHR1cmUvZjM0ODdkNDU3ZjkwNjJkNzg3YTNlNmNlMWM0NjY0YmY3NDAyZWM2N2RkMTExMjU2ZjE5YjM4Y2U0ZjY3MCJ9fX0="}]}}}`
2. Save the inventory by logging off or switching to a world with a different inventory.
#### The problem

This generates many errors in console even though the saving is done correctly and no items are lost.
#### Example error

```
[09:12:59 WARN]: [Multiverse-Inventories] Error while building configuration map.
java.lang.NullPointerException: null value in entry: skull-owner=null
        at com.google.common.collect.CollectPreconditions.checkEntryNotNull(CollectPreconditions.java:33) ~[creative.jar:git-Spigot-44c59bf-8f0f4ed]
        at com.google.common.collect.ImmutableMap.entryOf(ImmutableMap.java:135) ~[creative.jar:git-Spigot-44c59bf-8f0f4ed]
        at com.google.common.collect.ImmutableMap$Builder.put(ImmutableMap.java:206) ~[creative.jar:git-Spigot-44c59bf-8f0f4ed]
        at org.bukkit.craftbukkit.v1_8_R3.inventory.CraftMetaSkull.serialize(CraftMetaSkull.java:173) ~[creative.jar:git-Spigot-44c59bf-8f0f4ed]
        at org.bukkit.craftbukkit.v1_8_R3.inventory.CraftMetaItem.serialize(CraftMetaItem.java:850) ~[creative.jar:git-Spigot-44c59bf-8f0f4ed]
        at com.onarandombox.multiverseinventories.util.JsonConfiguration.buildMap(JsonConfiguration.java:55) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.JsonConfiguration.buildMap(JsonConfiguration.java:56) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.JsonConfiguration.buildMap(JsonConfiguration.java:46) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.JsonConfiguration.buildMap(JsonConfiguration.java:46) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.JsonConfiguration.saveToString(JsonConfiguration.java:32) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.EncodedJsonConfiguration.save(EncodedJsonConfiguration.java:30) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.data.FlatFilePlayerData.processProfileWrite(FlatFilePlayerData.java:243) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.data.FlatFilePlayerData.access$000(FlatFilePlayerData.java:30) [Multiverse-Inventories-2.5.jar:?]
        at com.onarandombox.multiverseinventories.util.data.FlatFilePlayerData$FileWriteThread.run(FlatFilePlayerData.java:209) [Multiverse-Inventories-2.5.jar:?]
```

**Comment by dumptruckman** [comment]
Please refer to #305 since it will probably be the fix for this.

**Comment by xtechgamer735** [comment]
Unable to reproduce this - most likely fixed in #305

**Comment by dumptruckman** [comment]
Closing, should be working in latest dev builds.
